superset-frontend/src/components/AsyncSelect/index.jsx

```diff
@@ -29,6 +29,7 @@ const propTypes = {
   mutator: PropTypes.func.isRequired,
   onAsyncError: PropTypes.func,
   value: PropTypes.oneOfType([
+    PropTypes.string,
```

Comment:
Adding `string` as one of the types for the `value` prop to fix the following error in the `QuerySearch` component:

```

Error: src/SqlLab/components/QuerySearch/index.tsx(193,13): error TS2322: Type 'string' is not assignable to type 'number | (number | null)[] | null | undefined'.
Error: src/SqlLab/components/QuerySearch/index.tsx(202,13): error TS2322: Type 'string' is not assignable to type 'number | (number | null)[] | null | undefined'.

```